docker/docker-compose-example.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  npps4:
+    image: MikuAuahDark/NPPS4:latest
```

Comment:
Can I change this example to not exposing my username?

Reply:
At that point I’d say to just delete the example, since (as arina said) you need to provide your username + repo name for users to be able to pull the image 

feel free to change username/repo name as needed